bundles/org.openhab.binding.valloxmv/src/main/java/org/openhab/binding/valloxmv/internal/ValloxMVWebSocket.java

```diff
@@ -387,15 +386,16 @@ public void onBinary(InputStream in) {
                         return;
                     }
                     // COMMAND_READ_DATA (250)
-                    /* Read data command is not implemented, response check is ready for future implementation
-                    if ((((iDataLength - 1) % 2) != 0) || (iDataLength < 5)) {
-                        logger.debug("Response corrupted, data length is wrong");
-                        return;
-                    }
-                    // number of data pairs (address, value) = (iDataLength - 3) / 2
-                    // First data pair (address, value) is in positions 2 and 3
-                    // (address, value) pairs could be put into array or hashmap
-                    */
```

Comment:
This shouldn't have been modified

Reply:
I have instructed the formatter of this.